Algorithms/Search/InterpolationSearch.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Algorithms.Search;
+
+/// <summary>
+/// An interpolation search requires a sorted array in order for it to work.
+/// </summary>
+public static class InterpolationSearch
+{
+    private static int[] values = { 1, 3, 5, 8, 10, 22, 31, 35, 37, 42, 51 };
+
+    /* L = Left most index
+     * R = Right most index
+     * Key = Value to find
+     */
+
+    /* Calculate an estimated index */
+    /* Formula: L + ((R-L) / (values[R] - values[L])) * (key - values[L]) */
+    /* Formula: 0 + ((10-0) / (51 - 1)) * (22 - 1) */
+
+    public static int Search()
```

Comment:
Where does it search? It doesn't have any parameters

Reply:
And what does it look for?

Reply:
This has been updated in my most recent commit.